cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -98,3 +100,71 @@ Then('I focus {int} row and focused row element has golden border on focus', (in
 });
+
+Then('I click on {string} header {int} times', (position, times) => {
+  for (let i = 0; i < times; i++) {
+    flatTableHeaderCells().find('div').eq(positionOfElement(position)).click();
+  }
+});
+
+When('{string} column is sorted in {string} order', (position, sortOrder) => {
+  const valueOne = 'Tyler Webb';
+  const valueTwo = 'Monty Parker';
+  const valueThree = 'Jason Atkinson';
+  const valueFour = 'Blake Sutton';
+  const totalOne = '3840';
+  const totalTwo = '1349';
+  const totalThree = '849';
+  const totalFour = '280';
+  if (position === 'first' && sortOrder === 'desc') {
+    icon().should('have.attr', 'data-element', 'sort_down')
+      .and('be.visible');
+    flatTableCell(positionOfElement('first')).should('have.text', valueOne)
+      .and('be.visible');
+    flatTableCell(positionOfElement('third')).should('have.text', valueTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fifth')).should('have.text', valueThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('seventh')).should('have.text', valueFour)
+      .and('be.visible');
+  } else if (position === 'first' && sortOrder === 'asc') {
+    icon().should('have.attr', 'data-element', 'sort_up')
+      .and('be.visible');
+    flatTableCell(positionOfElement('first')).should('have.text', valueFour)
+      .and('be.visible');
+    flatTableCell(positionOfElement('third')).should('have.text', valueThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fifth')).should('have.text', valueTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('seventh')).should('have.text', valueOne)
+      .and('be.visible');
+  } else if (position === 'second' && sortOrder === 'desc') {
+    icon().should('have.attr', 'data-element', 'sort_down')
+      .and('be.visible');
+    flatTableCell(positionOfElement('second')).should('have.text', totalOne)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fourth')).should('have.text', totalTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('sixth')).should('have.text', totalThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('eighth')).should('have.text', totalFour)
+      .and('be.visible');
+  } else {
+    icon().should('have.attr', 'data-element', 'sort_up')
+      .and('be.visible');
+    flatTableCell(positionOfElement('second')).should('have.text', totalFour)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fourth')).should('have.text', totalThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('sixth')).should('have.text', totalTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('eighth')).should('have.text', totalOne)
+      .and('be.visible');
+  }
+});
+
+Then('Flat table header has {string} color', (colorTheme) => {
+  flatTableHeaderCells().each(($el) => {
+    cy.wrap($el).should('have.css', 'background-color', colorTheme);
```

Comment:
do we need `cy.wrap()` here ? Or could just `expect()` will be enough ?

Reply:
we need `cy.wrap()` because the header has two cells and we need to check it separately.